### ISSUE TITLE
MultiSelect.Filterable a11y issues (activedescendant and initial DOM focus)

### ISSUE THREAD

**Issue by finken2** [carbon-design-system/carbon#7296]
Chrome, using the current accessibility checker plugin, there's two errors on the MultiSelect.Filterable component:
https://react.carbondesignsystem.com/?path=/story/multiselect--filterable

- Initial DOM focus on the combobox is not set on the text input when the combobox receives focus
- The <input> element using 'aria-activedescendant' set to "carbon-multiselect-example-item-1" does not have its 'tabindex' attribute value set to 0 or -1

This was reported during our AVT for Satellite, and is recreatable in the sandbox. Run the checker with the dropdown open and an option checked for the second one.

**Comment by tw15egan** [comment]
Thanks for reporting, @finken2! Going to close this out as a duplicate and track it over in https://github.com/carbon-design-system/carbon/issues/7331. @dakahn and I will take a look 👍
